setup.py

```diff
@@ -5,7 +5,7 @@
 
 setuptools.setup(
     name="prophecy-build-tool",
-    version="1.1.0.0",
+    version="1.1.0.1",
```

Comment:
@kirankaranth  why do we follow the revision version for a build?
Mostly it should be played with Minor or patch, anything lesser is very difficult to convey why we choose that to customers.

Reply:
Agree, I am also in support for Semver for Pbt